app/controllers/feedback_controller.rb

```diff
@@ -4,18 +4,18 @@ class FeedbackController < ApplicationController
   def new
     @journey = source
     @feedback = Feedback.new
-    @signed_out_provider_id = session.delete('signed_out_provider_id')
+    @signed_out = session.delete('signed_out')
+    @application_id = application_id
   end
 
-  def create
+  def create # rubocop:disable Metrics/AbcSize
```

Comment:
I'm not sure rubocop needs disabling here, I've just tried running it without and it doesn't give me a warning.

If you do do still get a warning then it might be worth refactoring some of the code into a new method to reduce the complexity rather than just disabling. You could do something like:
```
def initialize_feedback
  @feedback = Feedback.new(feedback_params)
  @feedback.originating_page = originating_page   
  @feedback.email = provider_email
end
```
and then call `initialize_feedback` in `create`